spec/index.bs

```diff
@@ -2870,16 +2870,19 @@ dictionary GPUBufferDescriptor : GPUObjectDescriptorBase {
         mapped range can be written/read to until it is unmapped.
 </dl>
 
-<div algorithm>
+<div algorithm data-timeline=device>
     <dfn abstract-op>validating GPUBufferDescriptor</dfn>(|device|, |descriptor|)
-        1. If |device| is lost return `false`.
-        1. If any of the bits of |descriptor|'s {{GPUBufferDescriptor/usage}} aren't present in
-            |device|'s [=allowed buffer usages=], return `false`.
-        1. If both the {{GPUBufferUsage/MAP_READ}} and {{GPUBufferUsage/MAP_WRITE}} bits of
-            |descriptor|'s {{GPUBufferDescriptor/usage}} attribute are set, return `false`.
-        1. If |descriptor|.{{GPUBufferDescriptor/size}} is greater than
-            |device|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.{{supported limits/maxBufferSize}}, return `false`.
-        1. Return `true`.
+    
+    [=Device timeline=] steps:
```

Comment:
We probably don't strictly need this? It's nice to have text corresponding to the styling, but there will always be such text in the *caller* of this function. So maybe we don't need it inside the function too?

Reply:
Fair enough. Done.